frontend/src/metabase/common/components/EntityPicker/components/EntityPickerModal/EntityPickerModal.unit.spec.tsx

```diff
@@ -294,7 +321,6 @@ describe("EntityPickerModal", () => {
         recentItems,
         defaultToRecentTab: false,
         initialValue: { model: "card" },
-        onConfirm: jest.fn(),
```

Comment:
i set a default for this prop so we could delete all these redundant props in each test